design/pv_backup_info.md

```diff
@@ -0,0 +1,144 @@
+# PersistentVolume backup information design
+
+## Abstract
+Create a new metadata file in the backup repository directory to store the backup including PVC and PV information. The information includes the way of backing up the PVC and PV data, snapshot information, and status. The needed snapshot status can also be recorded there, but the Velero-Native snapshot plugin doesn't provide a way to get the snapshot size from the API, so it's possible that not all snapshot size information is available.
```

Comment:
Is the metadata stored in backup folder or backup repository folder?

Reply:
The metadata file is stored under the backup sub-directory, same as the backup's log file <backup-name>-logs.gz

Reply:
Please update the line `Create a new metadata file in the backup repository directory` to correct the statement

Reply:
Modified.